lib/puppet_x/puppetlabs/scheduled_task/trigger.rb

```diff
@@ -357,7 +356,9 @@ def self.from_iTrigger(iTrigger)
         'minutes_duration'        => Duration.to_minutes(iTrigger.Repetition.Duration),
         'minutes_interval'        => Duration.to_minutes(iTrigger.Repetition.Interval),
         'flags'                   => trigger_flags,
-        'random_minutes_interval' => Trigger.string_to_int(iTrigger.Randomdelay)
+        # the V1 COM API always produces a value of 0 here and this is kept for
```

Comment:
This is the one reference I left @michaeltlombardi

Reply:
👍